delivery_cttexpress/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2022 Tecnativa - David Vidal
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Delivery CTT Express",
+    "summary": "Delivery Carrier implementation for CTT Express API",
+    "version": "16.0.1.0.0",
+    "category": "Delivery",
+    "website": "https://github.com/OCA/delivery-carrier",
+    "author": "Tecnativa, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "installable": True,
+    "depends": ["delivery_package_number", "delivery_state"],
+    "external_dependencies": {"python": ["zeep"]},
```

Comment:
Don't include it, as it's an standard requirement:

https://github.com/odoo/odoo/blob/16.0/requirements.txt#L50

Reply:
Done!